cf_remote/commands.py

```diff
@@ -606,7 +607,8 @@ def save(name, hosts, role):
     if "@" + name in state:
         print("Group '{}' already exists".format(name))
         return 1
-    group = {"meta": {"saved": True}}
+    current_time = datetime.now(timezone.utc)
+    group = {"meta": {"saved": True, "date": current_time.strftime("%y-%m-%dT%H:%M:%S%z")}}
```

Comment:
Maybe the same result, but I think it's better to just use `.isoformat()` for the format, rather than specifying something custom:

```suggestion
    current_time = datetime.now().astimezone().replace(microsecond=0).isoformat()
    group = {"meta": {"saved": True, "date": current_time)}}
```